iphone/TitaniumKit/TitaniumKit/Sources/API/TiRootViewController.m

```diff
@@ -1000,7 +1000,16 @@ - (BOOL)shouldRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOr
 - (void)adjustFrameForUpSideDownOrientation:(NSNotification *)notification
 {
   if ((![TiUtils isIPad]) && ([[UIApplication sharedApplication] statusBarOrientation] == UIInterfaceOrientationPortraitUpsideDown)) {
-    CGRect statusBarFrame = [[UIApplication sharedApplication] statusBarFrame];
```

Comment:
@hansemannn  This property is deprecated in iOS 13. Ideally it should not crash. But its better to use updated APIs

Reply:
Well, it did crash while testing my ti.scanner module :) Not the first time an API crashes although "only" being deprecated.